src/ralph/admin/mixins.py

```diff
@@ -57,6 +58,36 @@ def get_inline_media():
     )
 
 
+def initialize_search_form(model, context, fields_from_model=True):
+    """
+    Add to template's context extra variables (search_fields,
```

Comment:
-> Add extra variables (search_fields, search_url) which are used by search form to template's context.